streaming/java/streaming-api/src/main/java/io/ray/streaming/operator/chain/ChainedOperator.java

```diff
@@ -85,6 +86,23 @@ public Operator getTailOperator() {
     return tailOperator;
   }
 
+  @Override
+  public Serializable doCheckpoint() {
```

Comment:
Wiil be bettwen to name it as `saveCheckpoint` ? It sounds mot consitent with `loadCheckpoint `

Reply:
done.